pkg/api/mlflow/dao/models/app.go

```diff
@@ -0,0 +1,42 @@
+package models
+
+import (
+	"database/sql"
+	"database/sql/driver"
+	"encoding/json"
+
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type App struct {
```

Comment:
We try to avoid using shared objects like that, which contains `model` and `query` in the same object. can you please split them?

Reply:
There's not a query in this file, just a scan. See also api/mlflow/dao/models/run.go, which also has scan logic.

I was trying to get us a step closer to having a shared "models" package for both aim and mlflow by putting here, but i can just delete this and use pkg/database model?